tests/admin_views/tests.py

```diff
@@ -4675,6 +4675,7 @@ def test_save_continue_editing_button(self):
         self.assertRedirects(response, reverse('admin:auth_user_change', args=(new_user.pk,)))
         self.assertEqual(User.objects.count(), user_count + 1)
         self.assertTrue(new_user.has_usable_password())
+        self.assertFalse(new_user.has_disabled_password())
```

Comment:
Look in `tests/auth_tests` for a place to put the tests. `admin_views` tests are for `contrib.admin` rather than `contrib.auth`. Besides a test for the new `User.has_disabled_password()` method, there should also be a regression test for the change in `PasswordResetForm`.